app/src/main/java/com/dkanada/gramophone/ui/fragments/mainactivity/library/pager/AbsLibraryPagerRecyclerViewCustomGridSizeFragment.java

```diff
@@ -85,6 +94,13 @@ public void setAndSaveSortMethod(final String sortMethod) {
         invalidateAdapter();
     }
 
+    public void setAndSaveSortOrder(final String sortOrder) {
+        this.sortOrder = sortOrder;
+        saveSortOrder(sortOrder);
+        setSortOrder(sortOrder);
+        invalidateAdapter();
```

Comment:
I just installed the new version and this doesn't seem to be refreshing the list for the sort method.

Reply:
Mmm, I just tried and it worked as expected. What did you tried exactly?

Reply:
It was just taking longer than expected. When the method changes, it jumps to the top which looks like a refresh, and then actually replaces the data set a few seconds later. I'm currently making changes to the abstract classes to see if I can add a method for the queries there instead.